src/components/ContentEditable.tsx

```diff
@@ -67,9 +68,34 @@ const ContentEditable = React.memo(({ style, html, disabled, innerRef, ...props
     props.onChange(event)
   }
 
+  // eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
Please avoid disabling the linter. The function is not as self-evident as it seems; you can explain that it sets `text/em` so that the clipboard data can be later identified as coming from **em**.